### ISSUE TITLE
how can i get the label when running run_classifier.py

### ISSUE THREAD

**Issue by hxs91** [google-research/bert#264]
Yes, I have run the code successfully, but I want to know the label correspond to each column. and I use the following codes:
`with tf.gfile.GFile(output_predict_file, "w") as writer:

      tf.logging.info("***** Predict results *****")

      output_line = "\t".join(label for label in label_list) + "\n"

      writer.write(output_line)

      for prediction in result:

        print(prediction)

        output_line = "\t".join(
            str(class_probability) for class_probability in prediction) + "\n"

        writer.write(output_line)
`
but it seems wrong, because the performance is so poor when I calculate the precision according to the generated 'test_results.tsv'

**Comment by Colanim** [comment]
Add this to your `metric_fn` method :
```
def metric_fn(per_example_loss, label_ids, logits):       
        # Display labels and predictions
        concat1 = tf.contrib.metrics.streaming_concat(logits)
        concat2 = tf.contrib.metrics.streaming_concat(label_ids)
        
return {'pred': concat1, 'label_ids': concat2}
```

**Comment by hxs91** [comment]
I get the label by 
`
predict_label = tf.argmax(logits, axis=-1, output_type=tf.int32)
`
and the label is the same with my previous result...
so, it sames I am correct from the very first.
here comes the problems, my performance is so poor(precision: nearly 20%), and my task is a chinese text classififcation, the only reson I can guess is that: I did not perform a evaluation ?

**Comment by seven-linglx** [comment]
> I get the label by
> `predict_label = tf.argmax(logits, axis=-1, output_type=tf.int32)`
> and the label is the same with my previous result...
> so, it sames I am correct from the very first.
> here comes the problems, my performance is so poor(precision: nearly 20%), and my task is a chinese text classififcation, the only reson I can guess is that: my training is not enough ? 86000+ training instance, and the epoch is 3?

In addition, the learning rate is the key point also.

**Comment by hxs91** [comment]
It is the problem of shuffle. I did not shuffle my data.
